consensus/service/src/consensus_service.rs

```diff
@@ -704,24 +708,45 @@ impl<E: ConsensusEnclaveProxy, R: RaClient + Send + Sync + 'static> ConsensusSer
 
     /// Helper method for creating the get config json function needed by the GRPC admin service.
     fn create_get_config_json_fn(&self) -> GetConfigJsonFn {
+        let ledger_db = self.ledger_db.clone();
+        let byzantine_ledger = self.byzantine_ledger.clone();
         let config = self.config.clone();
         Arc::new(move || {
+            let block_height = ledger_db.num_blocks().unwrap_or(0);
+            let mut sync_status = "synced";
+            let mut peer_block_height: u64 = 0;
+            let byzantine_ledger = byzantine_ledger
+                .lock()
+                .expect("Could not get byzantine ledger.");
+            if let Some(byzantine_ledger) = &*byzantine_ledger {
+                if byzantine_ledger.is_behind() {
+                    sync_status = "catchup";
+                };
+                peer_block_height = byzantine_ledger.highest_peer_block();
+            }
             Ok(json!({
                 "config": {
                     "public_key": config.node_id().public_key,
                     "peer_responder_id": config.peer_responder_id,
                     "client_responder_id": config.client_responder_id,
                     "message_pubkey": config.msg_signer_key.public_key(),
                     "network": config.network_path,
-                    "ias_api_key": config.ias_api_key,
-                    "ias_spid": config.ias_spid,
                     "peer_listen_uri": config.peer_listen_uri,
                     "client_listen_uri": config.client_listen_uri,
                     "admin_listen_uri": config.admin_listen_uri,
                     "ledger_path": config.ledger_path,
                     "scp_debug_dump": config.scp_debug_dump,
                 },
                 "network": config.network(),
+                "status": {
+                    "block_height": block_height,
+                    "version": VERSION,
+                    "broadcast_peer_count": config.network().broadcast_peers.len(),
+                    "known_peer_count": config.network().known_peers.map(|x| x.len()),
+                    "sync_status": sync_status,
+                    "blocks_behind": std::cmp::min(peer_block_height - block_height, 0),
+                    "latest_block_hash": ledger_db.get_block(block_height - 1).map(|x| x.id).ok(),
```

Comment:
How does serializing of an Option looks like in JSON?

Reply:
If it's None, it's a json "null" - my intent here was to provide the hash unless it errored, in which cause display "null."